packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -101,45 +101,92 @@ var config = {
 
-![Displays the registration objects in JavaScript to remove](/img/SIW_Upgrade_Config_Change1.png)
+```JavaScript
+var signIn = new OktaSignIn({
+  baseUrl: 'https://{yourOktaDomain}',
+  registration: {
+    preSubmit: (postData, onSuccess, onFailure) => {
+        // handle preSubmit callback
+        onSuccess(postData);
+    },
+    postSubmit: (response, onSuccess, onFailure) => {
+        // handle postsubmit callback
+        onSuccess(response);
+    }
+  },
+  features: {
+    // Used to enable registration feature on the widget
+    // https://github.com/okta/okta-signin-widget#feature-flags
+    registration: true
+  }
+})
+```
+
+> **Note**: We have not included the `parseSchema` method in the above `registration` object; the ability to include additional schemas is no longer supported.
 
 ### IdP Discovery
 
 IdP Discovery enables you to route users to different third-party IdPs that are connected to your Okta org. Users can federate back into the primary org after authenticating at the IdP. This feature still functions, but you no longer need to enable the link for users to initialize the route. Instead, you can configure a routing rule with the application context.
 
-You can remove the `idpDiscovery` JavaScript object in the widget as follows.
+Remove the `idpDiscovery` property shown below:
 
-![Displays the IDP Discovery object in JavaScript to remove](/img/SIW_Upgrade_Config_Change2.png)
+```JavaScript
+features: {
+  idpDiscovery: true
+}
+```
 
 ### OpenID Connect/social authentication
 
-When [external Identity Providers (IdPs)](/docs/guides/identity-providers/) are used for Social Login in Classic Engine, the supported IdPs (such as Google, Facebook, Apple, Microsoft, and so on) are declared with a `type` property. These supported IdPs get distinct styling and default i18n text, while other custom entries receive general styling and require you to provide text. Each IdP can have additional CSS classes added through an optional `className` property. This feature still functions in Identity Engine. However, it's no longer the preferred method to enable the link for users to initialize the route. You can replace this method by configuring a routing rule with the application context. See [Okta Sign-In Widget with an external Identity Provider](/docs/guides/add-an-external-idp/openidconnect/main/#okta-sign-in-widget).
-
-You no longer need the `idps` JavaScript object and `className` property in the widget, and you can remove them as follows.
+You no longer require the `idps` JavaScript object and `className` property in the widget (shown below for reference), and you can remove them.  This is optional as the Okta Identity Engine will now include IdPs based on the routing rules.
 
-![Displays the IdPs object and className property in JavaScript to remove](/img/SIW_Upgrade_Config_Change3.png)
+```JavaScript
+idps: [
+  { type: 'GOOGLE', id: '0oagjkh' },
+  { type: 'FACEBOOK', id: '0oagjkh' },
+]
+```
```

Comment:
Add this after the snippet?

The widget displays IdPs based on Identity Engine routing rules.

Reply:
Again, pretty much did what you suggested just slightly differently. Really appreciate the idea.